frontend/javascripts/libs/input.ts

```diff
@@ -113,7 +157,14 @@ export class InputKeyboardNoLoop {
       },
       _.noop,
     ];
-    KeyboardJS.bind(...binding);
+    if (isExtendedCommand) {
+      KeyboardJS.withContext("extended", () => {
+        KeyboardJS.bind(...binding);
+      });
+    }
+    KeyboardJS.withContext("default", () => {
+      KeyboardJS.bind(...binding);
+    });
```

Comment:
why isn't the default case in an else-clause? aren't extended shortcuts now defined both in the extended and in the default context?

Reply:
oh right, that is actually important to have the else. I somehow forgot this :see_no_evil:. It is fixed now.